.ci/merge-fixes.sh

```diff
@@ -67,7 +68,7 @@ for REPO in ${SAGE_CI_FIXES_FROM_REPOSITORIES:-sagemath/sage}; do
                         git am --signoff --show-current-patch=diff
                         echo "--------------------------------------------------------------------8<-----------------------------"
                         echo "::endgroup::"
-                        echo "Failure applying $PULL_URL as a patch, resetting"
+                        echo "Failure applying $PULL_SHORT as a patch, resetting"
```

Comment:
I like the full URL since it easy to open the corresponding PR. Either way, these changes are unrelated to the actual goal of the PR and thus should be moved to somewhere else.

Reply:
The full URL appears just above. The improvement is that the repeated one is shortened so that there is not a second, duplicate link.

I'll not open a separate PR for this.

Reply:
Why not? This is clearly not connected to main theme of this PR.

Reply:
You've just finished the review of this one line change, no?

Reply:
No, and there are also more changes here that seem unrelated.

Reply:
You commented, I responded to your comment. What's missing for the review of this one line change.

Reply:
The rules of review, spelled out for you: 
- Provide the necessary information. 
- Do not use vagueness as a tool.
- Do not ask for information that has already been provided.